index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <header id="header">
+      <div class="title">
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="" id="header-img">
+      </div>
+      <nav id="nav-bar"><ul>
+        <li class="nav-link"><a href="#services">Features</a></li>
+        <li class="nav-link"><a href="#">How It works</a></li>
+        <li class="nav-link"><a href="#">Pricing</a></li>
+      </ul></nav>
+    </header>
+    <main>
+      <section id="header-content">
+        <h2>Handcrafted, home-made masterpieces</h2>
+        <form action="https://www.freecodecamp.com/email-submit" id="form">
+          <input type="email" id="email" placeholder="Enter your email address"><br>
+          <input type="submit" id="submit" value="Get Started">
+        </form>
+      </section>
+    </main>
+
+
+    <div id="services">
+      <ul>
+        <li>          
+          <h2><img src="./nut.svg" alt="">Premium Materials</h2>
+          <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase. </p>
+        </li>
+        <li>          
+          <h2><img src="./delivery.svg" alt="">Fast Shipping</h2>
+          <p>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.  </p>
+        </li>
+        <li>
+          <h2><img src="./pencil-edit.svg" alt="">Quality Assurance</h2>
+          <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument. </p>
+        </li>
+          
+      </ul>
+    </div>
```

Comment:
Please wrap these lines of code in a section tag. Using semantic tags accurately describes the content that is on the page.